mods/cnc/tilesets/jungle.yaml

```diff
@@ -449,8 +449,8 @@ Templates:
 		Size: 2,2
 		Category: Cliffs
 		Tiles:
-			0: Clear
-			1: Clear
+			0: Rock
```

Comment:
Doesn't this want to be `Rough` here and below?
![tdcliff2](https://cloud.githubusercontent.com/assets/7704140/12011112/44573808-acc2-11e5-8715-27faf3ba5928.png)
(As you made a similiar looking spot `Rough` on tile number 35.)